joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/cpg/MethodCpgTests.scala

```diff
@@ -0,0 +1,25 @@
+package io.joern.pysrc2cpg.cpg
+
+import io.joern.pysrc2cpg.Py2CpgTestContext
+import io.shiftleft.semanticcpg.language._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+class MethodCpgTests extends AnyFreeSpec with Matchers {
+  "A method" - {
+
+    lazy val cpg = Py2CpgTestContext.buildCpg(
+      """
+        |def method():
+        |   pass
+        |""".stripMargin,
+      "a/b.py"
```

Comment:
Should this not be something like `Seq("a", "b.py").mkString(File.separator)`

Reply:
It should not because the OS file handling layer will correctly handle the slash as file separator even on windows.